### DISCUSSION TITLE
Hp Probook 6470b - glitches on booting Ventura installer

### DISCUSSION THREAD

**Discussion by spn0381** [chris1111/HP-Probook-EliteBook-Package-Creator#68] [General]
Hi, I have an HP Probook 6470b with big sur installed (with an old Chris1111 OC package) and I'm going to trying to update it with a fresh ventura...

I've 8gb ram, HD4000, Intel I5. Bios F74

during the boot, The screen glitch and I cannot see / recognize anything in the screen. Any Idea how I can solve this problem?

![IMG_3418](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator/assets/17476534/1c8f1e59-57c5-40dc-8a07-b7aac85caf89)
![IMG_3417](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator/assets/17476534/97ca092c-3d22-4eb1-9700-ae9eb34df05f)
![IMG_3416](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator/assets/17476534/742ea5cb-5c20-4b66-a49d-4ba9198e9aa6)
![IMG_3414](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator/assets/17476534/e5a327e4-b933-4113-b9fc-12606bef7892)

thank you in advance for your help

**Comment by spn0381** [comment]
https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator-OC/discussions

I don't know how to move the discussion...

**Comment by chris1111** [comment]
BIOS - / ADVANCED / BOOT OPTION / BOOT MODE / `UEFI Hybrid With CSM`

**Comment by spn0381** [comment]
Thank you for the fast support! I’m installing !!!
